portfolio/src/main/webapp/script.js

```diff
@@ -28,7 +28,11 @@ function addRandomQuote() {
 }
 // fetch message
 async function getMessage() {
-  const response = await fetch('/data');
-  const message = await response.text();
-  document.getElementById('message-container').innerText = "This is a message worthy of getting";
+  fetch('/index')  // sends a request to /my-data-url
```

Comment:
Out of curiosity, why was this changed to '/index'? It looks like '/data' is the servlet that's implemented in DataServlet.java.

Reply:
I was trying all sorts of things to get it to work. I had tried it with data and it had failed